### ISSUE TITLE
[ENHANCEMENT] 你好，fp8有计划支持吗

### ISSUE THREAD

**Issue by zkyue** [bytedance/flux#48]

**Comment by wenlei-bao** [comment]
@zkyue Yes, fp8 support is on the way. And we will release it in future.

**Comment by zkyue** [comment]
> @zkyue Yes, fp8 support is on the way. And we will release it in future.

Thank you for your reply. I am now looking to apply Flux in my project to perform FP8 GEMM on the H100. What work still needs to be done with Flux at the moment? Is it just a matter of adding a data type like supporting FP16, or are there some underlying tasks that need to be addressed? How is the progress of your project, and when can we expect the next version to be released?

**Comment by houqi** [comment]
> > [@zkyue](https://github.com/zkyue) Yes, fp8 support is on the way. And we will release it in future.
> 
> Thank you for your reply. I am now looking to apply Flux in my project to perform FP8 GEMM on the H100. What work still needs to be done with Flux at the moment? Is it just a matter of adding a data type like supporting FP16, or are there some underlying tasks that need to be addressed? How is the progress of your project, and when can we expect the next version to be released?

FP8 is supported. checkout the lastest main or download the latest wheel and have a try.
